### PR TITLE
Remove cxxopt from dr tests suite

### PR DESCRIPTION
**Pull request by mateuszpn**

### CODE REVIEW

File: test/distributed-ranges/shp/xhp-tests.hpp [hunk 1]
```diff
@@ -3,23 +3,19 @@
 // SPDX-License-Identifier: BSD-3-Clause
 #pragma once
 
-#include "cxxopts.hpp"
+// #include "cxxopts.hpp"
```
Comment (lslusarczyk) [inline test/distributed-ranges/shp/xhp-tests.hpp:6]: remove instead of commenting out

**Review by lslusarczyk — CHANGES_REQUESTED** [review]
some comments

Since changes shown in github are not strictly the changes you're about to commit I advice to check diff in cmdline by
`git diff origin/distributed-ranges`

Begore doing this please checkout latest brancgh as I've resolved  recent conflicts there.

File: include/oneapi/dpl/internal/distributed_ranges_impl/detail/enumerate.hpp [hunk 2]
```diff
@@ -40,10 +40,10 @@ class enumerate_adapter_closure {
     }
   }
 
-  template <rng::viewable_range R>
```
Comment (lslusarczyk) [inline include/oneapi/dpl/internal/distributed_ranges_impl/detail/enumerate.hpp:43]: revert, this is only formatting change

**Review by lslusarczyk — CHANGES_REQUESTED** [review]
seems diff shown in github is correct one now

File: test/distributed-ranges/include/common-tests.hpp [hunk 3]
```diff
@@ -141,21 +166,25 @@ auto equal_message(rng::range auto &&ref, rng::range auto &&actual,
   }
   return drfmt::format("\n{}"
                      "    ref:    {}\n"
-                     "    actual: {}\n  ",
```
[the remark this answers is not in the document]
Comment (mateuszpn) [reply test/distributed-ranges/include/common-tests.hpp:166]: These changes were accidental. After manual restoring of relevant snippets, formating may be slightly different, as now it follows LLVM rules, as defined by VC Code

**Comment by lslusarczyk** [comment]
Removed formatting changes, removed cxxopts header checker from cmake.

**Pull request merged**